orchestrator/redisClientWrapper.js

```diff
@@ -0,0 +1,61 @@
+/* jslint node: true */
+"use strict";
+var axios = require("axios");
+
+var config = require('./config');
+var auth = require('./auth');
+
+/**
+ * Client for REDIS database
+ */
+class ClientWrapper {
+
+  constructor(client) {
+    this.client = client;
+  }
+  /**
+   * Returns template list of a device. Search for it on cache, if it's not, requests device manager
+   * and then stores on cache.
+   * @param deviceid 
+   */
+
+  getTemplateList(deviceid, tenant) {
```

Comment:
I will modify to consider the tenant to store/retrieve the data, because, even if the deviceId is unique for all tenants, it will cover all the cases.